roles/openshift_cfme/files/openshift_cfme.exports

```diff
@@ -0,0 +1,3 @@
+/exports/miq-pv01 *(rw,no_root_squash,no_wdelay)
+/exports/miq-pv02 *(rw,no_root_squash,no_wdelay)
+/exports/miq-pv03 *(rw,no_root_squash,no_wdelay)
```

Comment:
wrt: external NFS. I see that as an option in the future. For now that will not be part of the proof of concept.